extern/storage-sealing/precommit_policy_test.go

```diff
@@ -38,21 +58,49 @@ func fakePieceCid(t *testing.T) cid.Cid {
 }
 
 func TestBasicPolicyEmptySector(t *testing.T) {
-	policy := sealing.NewBasicPreCommitPolicy(&fakeChain{
-		h: abi.ChainEpoch(55),
-	}, 10, 0)
+	cfg := fakeConfigGetter(nil)
+	h := abi.ChainEpoch(55)
+	pBoundary := abi.ChainEpoch(0)
+	pBuffer := abi.ChainEpoch(2)
+	pcp := sealing.NewBasicPreCommitPolicy(&fakeChain{h: h}, cfg, pBoundary, pBuffer)
+	exp, err := pcp.Expiration(context.Background())
+
+	require.NoError(t, err)
+
+	// as set when there are no deal pieces
+	expected := h + policy.GetMaxSectorExpirationExtension() - (pBuffer * 2)
+	// as set just before returning within Expiration()
+	expected += miner.WPoStProvingPeriod - (expected % miner.WPoStProvingPeriod) + pBoundary - 1
+	assert.Equal(t, int(expected), int(exp))
+}
```

Comment:
The math is a little obtuse here so I put literal `expected`s for both paths (zero config for CC and non-CC sectors) as well as show how the custom value is applied. I kept the later units with literals in the assert and simply updated the values.